project/manager.go

```diff
@@ -39,10 +40,22 @@ func NewManager(chainEndpoint, contractAddress, projectFileDirectory string) (*M
 		if err != nil {
 			return nil, errors.Wrapf(err, "read project file %s failed", f.Name())
 		}
-		p := Project{}
-		if err := json.Unmarshal(data, &p); err != nil {
-			return nil, errors.Wrapf(err, "unmarshal project file %s failed", f.Name())
+
+		projectID, err := strconv.ParseUint(f.Name(), 10, 64)
```

Comment:
maybe config file add .json suffix will better

Reply:
I think just project ID is enough. Because the file is generated by wsctl, the developer do not need to edit it. if he care about the content type of file, he can read it.